projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/monitoring/DataJobMonitor.java

```diff
@@ -195,7 +196,7 @@ void recordJobExecutionStatus(KubernetesService.JobExecution jobStatus) {
         final DataJob dataJob = dataJobOptional.get();
 
         // Update the job execution and the last execution state
-        jobExecutionService.updateJobExecution(dataJob, jobStatus, executionResult)
+        jobExecutionService.updateJobExecution(dataJob, jobStatus, executionResult, containerTerminationReason)
```

Comment:
You don't need an additional parameter since we already have a parameter jobStatus that contains the termination reason.

```suggestion
        jobExecutionService.updateJobExecution(dataJob, jobStatus, executionResult)
```